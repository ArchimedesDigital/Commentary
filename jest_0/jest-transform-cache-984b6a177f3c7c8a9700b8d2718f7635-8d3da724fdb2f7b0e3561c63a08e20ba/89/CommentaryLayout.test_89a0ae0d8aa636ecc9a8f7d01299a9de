/* istanbul ignore next */'use strict';var /* istanbul ignore next */_react = require('react'); /* istanbul ignore next */var _react2 = _interopRequireDefault(_react);
var /* istanbul ignore next */_reactTestRenderer = require('react-test-renderer'); /* istanbul ignore next */var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);


var /* istanbul ignore next */_CommentaryLayout = require('./CommentaryLayout'); /* istanbul ignore next */var _CommentaryLayout2 = _interopRequireDefault(_CommentaryLayout);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}

describe('CommentaryLayout', function () {
	it('renders correctly', function () {

		var tree = /* istanbul ignore next */_reactTestRenderer2.default.
		create( /* istanbul ignore next */_react2.default.createElement( /* istanbul ignore next */_CommentaryLayout2.default, /* istanbul ignore next */{ isTest: /* istanbul ignore next */true })).
		toJSON();
		expect(tree).toMatchSnapshot();
	});
}); // component: