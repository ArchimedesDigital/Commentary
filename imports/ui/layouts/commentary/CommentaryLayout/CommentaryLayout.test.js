import React from 'react';
import renderer from 'react-test-renderer';

// component:
import CommentaryLayout from './CommentaryLayout';

describe('CommentaryLayout', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(<CommentaryLayout />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
