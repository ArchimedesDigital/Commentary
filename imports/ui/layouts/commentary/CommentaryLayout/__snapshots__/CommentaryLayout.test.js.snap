// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentaryLayout renders correctly 1`] = `
<div>
  <div
    className="chs-layout commentary-layout"
  >
    <div>
      <div
        className="md-sidenav-left"
        style={undefined}
      >
        <div
          className={undefined}
          onTouchTap={[Function]}
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
              "backgroundColor": "rgba(0, 0, 0, 0.54)",
              "height": "100%",
              "left": "-100%",
              "muiPrepared": true,
              "opacity": 0,
              "pointerEvents": "none",
              "position": "fixed",
              "top": 0,
              "transform": "translateZ(0)",
              "transition": "left 0ms cubic-bezier(0.23, 1, 0.32, 1) 400ms, opacity 400ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "width": "100%",
              "willChange": "opacity",
              "zIndex": 1200,
            }
          }
        />
        <div
          className={undefined}
          style={
            Object {
              "WebkitOverflowScrolling": "touch",
              "WebkitTapHighlightColor": "rgba(0,0,0,0)",
              "backgroundColor": "#ffffff",
              "borderRadius": "0px",
              "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                     0 3px 10px rgba(0, 0, 0, 0.23)",
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "fontFamily": "Roboto, sans-serif",
              "height": "100%",
              "left": 0,
              "muiPrepared": true,
              "overflow": "auto",
              "position": "fixed",
              "top": 0,
              "transform": "translate(-266px, 0)",
              "transition": "transform 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "width": 256,
              "zIndex": 1300,
            }
          }
        >
          <div
            className="sidenav-top"
          >
            <a
              href="/profile"
            >
              <span
                className="user-fullname"
              >
                
              </span>
            </a>
          </div>
          
          <a
            className="menu-item"
            href="/"
            onClick={[Function]}
            style={
              Object {
                "display": "block",
                "fontSize": "16px",
                "padding": "15px 15px",
                "width": "100%",
              }
            }
            target={undefined}
          >
            Home
          </a>
          <hr
            style={
              Object {
                "backgroundColor": "#e0e0e0",
                "border": "none",
                "height": 1,
                "margin": 0,
                "marginLeft": 0,
                "marginTop": -1,
                "muiPrepared": true,
              }
            }
          />
          <div>
            <a
              className="menu-item"
              href="/sign-in"
              onClick={[Function]}
              style={
                Object {
                  "display": "block",
                  "fontSize": "16px",
                  "padding": "15px 15px",
                  "width": "100%",
                }
              }
              target={undefined}
            >
              Sign in
            </a>
          </div>
        </div>
      </div>
      <div
        className="search-tools-drawer"
        style={
          Object {
            "backgroundColor": "#ddd",
          }
        }
      >
        <div
          className={undefined}
          onTouchTap={[Function]}
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
              "backgroundColor": "rgba(0, 0, 0, 0.54)",
              "height": "100%",
              "left": "-100%",
              "muiPrepared": true,
              "opacity": 0,
              "pointerEvents": "none",
              "position": "fixed",
              "top": 0,
              "transform": "translateZ(0)",
              "transition": "left 0ms cubic-bezier(0.23, 1, 0.32, 1) 400ms, opacity 400ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "width": "100%",
              "willChange": "opacity",
              "zIndex": 1200,
            }
          }
        />
        <div
          className={undefined}
          style={
            Object {
              "WebkitOverflowScrolling": "touch",
              "WebkitTapHighlightColor": "rgba(0,0,0,0)",
              "backgroundColor": "#ffffff",
              "borderRadius": "0px",
              "boxShadow": "0 3px 10px rgba(0, 0, 0, 0.16),
                     0 3px 10px rgba(0, 0, 0, 0.23)",
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "fontFamily": "Roboto, sans-serif",
              "height": "100%",
              "left": "auto",
              "muiPrepared": true,
              "overflow": "auto",
              "position": "fixed",
              "right": 0,
              "top": 0,
              "transform": "translate(410px, 0)",
              "transition": "transform 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "width": 400,
              "zIndex": 1300,
            }
          }
        >
          <div
            className="search-tool text-search text-search--drawer"
          >
            
          </div>
          <div
            className="search-tool-card"
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                "backgroundColor": "#ffffff",
                "borderRadius": "2px",
                "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
                       0 1px 4px rgba(0, 0, 0, 0.12)",
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "fontFamily": "Roboto, sans-serif",
                "muiPrepared": true,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "zIndex": 1,
              }
            }
          >
            <div
              style={
                Object {
                  "paddingBottom": 0,
                }
              }
            >
              <div
                className="card-header"
                onTouchTap={[Function]}
                style={
                  Object {
                    "boxSizing": "border-box",
                    "cursor": "pointer",
                    "fontFamily": "Proxima N W01 At Smbd",
                    "fontSize": "14px",
                    "fontWeight": "bold",
                    "muiPrepared": true,
                    "padding": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "whiteSpace": "nowrap",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "muiPrepared": true,
                      "paddingRight": "90px",
                      "verticalAlign": "top",
                      "whiteSpace": "normal",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "display": "block",
                        "fontSize": 15,
                        "muiPrepared": true,
                      }
                    }
                  >
                    Work
                  </span>
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                        "display": "block",
                        "fontSize": 14,
                        "muiPrepared": true,
                      }
                    }
                  />
                </div>
                <button
                  disabled={false}
                  href={undefined}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onMouseOut={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  onTouchTap={[Function]}
                  style={
                    Object {
                      "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                      "background": "none",
                      "border": 10,
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "cursor": "pointer",
                      "display": "inline-block",
                      "fontFamily": "Roboto, sans-serif",
                      "fontSize": 0,
                      "fontWeight": "inherit",
                      "height": 48,
                      "margin": "auto",
                      "muiPrepared": true,
                      "outline": "none",
                      "overflow": "visible",
                      "padding": 12,
                      "position": "absolute",
                      "right": 4,
                      "textDecoration": "none",
                      "top": 0,
                      "transform": "translate(0, 0)",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "verticalAlign": null,
                      "width": 48,
                    }
                  }
                  tabIndex={0}
                  type="button"
                >
                  <div
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchStart={[Function]}
                  >
                    <svg
                      onMouseEnter={[Function]}
                      onMouseLeave={[Function]}
                      style={
                        Object {
                          "color": "rgba(0, 0, 0, 0.87)",
                          "display": "inline-block",
                          "fill": "currentColor",
                          "height": 24,
                          "muiPrepared": true,
                          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                          "userSelect": "none",
                          "width": 24,
                        }
                      }
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M7.41 7.84L12 12.42l4.59-4.58L18 9.25l-6 6-6-6z"
                      />
                    </svg>
                  </div>
                </button>
              </div>
            </div>
          </div>
          <div
            className="search-tool-card"
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                "backgroundColor": "#ffffff",
                "borderRadius": "2px",
                "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
                       0 1px 4px rgba(0, 0, 0, 0.12)",
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "fontFamily": "Roboto, sans-serif",
                "muiPrepared": true,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "zIndex": 1,
              }
            }
          >
            <div
              style={
                Object {
                  "paddingBottom": 0,
                }
              }
            >
              <div
                className="card-header "
                style={
                  Object {
                    "boxSizing": "border-box",
                    "fontFamily": "Proxima N W01 At Smbd",
                    "fontSize": "14px",
                    "fontWeight": "bold",
                    "muiPrepared": true,
                    "padding": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "whiteSpace": "nowrap",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "muiPrepared": true,
                      "paddingRight": "90px",
                      "verticalAlign": "top",
                      "whiteSpace": "normal",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "display": "block",
                        "fontSize": 15,
                        "muiPrepared": true,
                      }
                    }
                  >
                    Book
                  </span>
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                        "display": "block",
                        "fontSize": 14,
                        "muiPrepared": true,
                      }
                    }
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="search-tool-card"
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                "backgroundColor": "#ffffff",
                "borderRadius": "2px",
                "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
                       0 1px 4px rgba(0, 0, 0, 0.12)",
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "fontFamily": "Roboto, sans-serif",
                "muiPrepared": true,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "zIndex": 1,
              }
            }
          >
            <div
              style={
                Object {
                  "paddingBottom": 0,
                }
              }
            >
              <div
                className="card-header "
                style={
                  Object {
                    "boxSizing": "border-box",
                    "fontFamily": "Proxima N W01 At Smbd",
                    "fontSize": "14px",
                    "fontWeight": "bold",
                    "muiPrepared": true,
                    "padding": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "whiteSpace": "nowrap",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "muiPrepared": true,
                      "paddingRight": "90px",
                      "verticalAlign": "top",
                      "whiteSpace": "normal",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "display": "block",
                        "fontSize": 15,
                        "muiPrepared": true,
                      }
                    }
                  >
                    Line Number
                  </span>
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                        "display": "block",
                        "fontSize": 14,
                        "muiPrepared": true,
                      }
                    }
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            className="search-tool-card"
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                "backgroundColor": "#ffffff",
                "borderRadius": "2px",
                "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
                       0 1px 4px rgba(0, 0, 0, 0.12)",
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "fontFamily": "Roboto, sans-serif",
                "muiPrepared": true,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "zIndex": 1,
              }
            }
          >
            <div
              style={
                Object {
                  "paddingBottom": 0,
                }
              }
            >
              <div
                className="card-header"
                onTouchTap={[Function]}
                style={
                  Object {
                    "boxSizing": "border-box",
                    "cursor": "pointer",
                    "fontFamily": "Proxima N W01 At Smbd",
                    "fontSize": "14px",
                    "fontWeight": "bold",
                    "muiPrepared": true,
                    "padding": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "whiteSpace": "nowrap",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "muiPrepared": true,
                      "paddingRight": "90px",
                      "verticalAlign": "top",
                      "whiteSpace": "normal",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "display": "block",
                        "fontSize": 15,
                        "muiPrepared": true,
                      }
                    }
                  >
                    Keywords
                  </span>
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                        "display": "block",
                        "fontSize": 14,
                        "muiPrepared": true,
                      }
                    }
                  />
                </div>
                <button
                  disabled={false}
                  href={undefined}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onMouseOut={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  onTouchTap={[Function]}
                  style={
                    Object {
                      "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                      "background": "none",
                      "border": 10,
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "cursor": "pointer",
                      "display": "inline-block",
                      "fontFamily": "Roboto, sans-serif",
                      "fontSize": 0,
                      "fontWeight": "inherit",
                      "height": 48,
                      "margin": "auto",
                      "muiPrepared": true,
                      "outline": "none",
                      "overflow": "visible",
                      "padding": 12,
                      "position": "absolute",
                      "right": 4,
                      "textDecoration": "none",
                      "top": 0,
                      "transform": "translate(0, 0)",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "verticalAlign": null,
                      "width": 48,
                    }
                  }
                  tabIndex={0}
                  type="button"
                >
                  <div
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchStart={[Function]}
                  >
                    <svg
                      onMouseEnter={[Function]}
                      onMouseLeave={[Function]}
                      style={
                        Object {
                          "color": "rgba(0, 0, 0, 0.87)",
                          "display": "inline-block",
                          "fill": "currentColor",
                          "height": 24,
                          "muiPrepared": true,
                          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                          "userSelect": "none",
                          "width": 24,
                        }
                      }
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M7.41 7.84L12 12.42l4.59-4.58L18 9.25l-6 6-6-6z"
                      />
                    </svg>
                  </div>
                </button>
              </div>
            </div>
          </div>
          <div
            className="search-tool-card"
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                "backgroundColor": "#ffffff",
                "borderRadius": "2px",
                "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
                       0 1px 4px rgba(0, 0, 0, 0.12)",
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "fontFamily": "Roboto, sans-serif",
                "muiPrepared": true,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "zIndex": 1,
              }
            }
          >
            <div
              style={
                Object {
                  "paddingBottom": 0,
                }
              }
            >
              <div
                className="card-header"
                onTouchTap={[Function]}
                style={
                  Object {
                    "boxSizing": "border-box",
                    "cursor": "pointer",
                    "fontFamily": "Proxima N W01 At Smbd",
                    "fontSize": "14px",
                    "fontWeight": "bold",
                    "muiPrepared": true,
                    "padding": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "whiteSpace": "nowrap",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "muiPrepared": true,
                      "paddingRight": "90px",
                      "verticalAlign": "top",
                      "whiteSpace": "normal",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "display": "block",
                        "fontSize": 15,
                        "muiPrepared": true,
                      }
                    }
                  >
                    Key Ideas
                  </span>
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                        "display": "block",
                        "fontSize": 14,
                        "muiPrepared": true,
                      }
                    }
                  />
                </div>
                <button
                  disabled={false}
                  href={undefined}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onMouseOut={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  onTouchTap={[Function]}
                  style={
                    Object {
                      "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                      "background": "none",
                      "border": 10,
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "cursor": "pointer",
                      "display": "inline-block",
                      "fontFamily": "Roboto, sans-serif",
                      "fontSize": 0,
                      "fontWeight": "inherit",
                      "height": 48,
                      "margin": "auto",
                      "muiPrepared": true,
                      "outline": "none",
                      "overflow": "visible",
                      "padding": 12,
                      "position": "absolute",
                      "right": 4,
                      "textDecoration": "none",
                      "top": 0,
                      "transform": "translate(0, 0)",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "verticalAlign": null,
                      "width": 48,
                    }
                  }
                  tabIndex={0}
                  type="button"
                >
                  <div
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchStart={[Function]}
                  >
                    <svg
                      onMouseEnter={[Function]}
                      onMouseLeave={[Function]}
                      style={
                        Object {
                          "color": "rgba(0, 0, 0, 0.87)",
                          "display": "inline-block",
                          "fill": "currentColor",
                          "height": 24,
                          "muiPrepared": true,
                          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                          "userSelect": "none",
                          "width": 24,
                        }
                      }
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M7.41 7.84L12 12.42l4.59-4.58L18 9.25l-6 6-6-6z"
                      />
                    </svg>
                  </div>
                </button>
              </div>
            </div>
          </div>
          <div
            className="search-tool-card"
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                "backgroundColor": "#ffffff",
                "borderRadius": "2px",
                "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
                       0 1px 4px rgba(0, 0, 0, 0.12)",
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "fontFamily": "Roboto, sans-serif",
                "muiPrepared": true,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "zIndex": 1,
              }
            }
          >
            <div
              style={
                Object {
                  "paddingBottom": 0,
                }
              }
            >
              <div
                className="card-header"
                onTouchTap={[Function]}
                style={
                  Object {
                    "boxSizing": "border-box",
                    "cursor": "pointer",
                    "fontFamily": "Proxima N W01 At Smbd",
                    "fontSize": "14px",
                    "fontWeight": "bold",
                    "muiPrepared": true,
                    "padding": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "whiteSpace": "nowrap",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "muiPrepared": true,
                      "paddingRight": "90px",
                      "verticalAlign": "top",
                      "whiteSpace": "normal",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "display": "block",
                        "fontSize": 15,
                        "muiPrepared": true,
                      }
                    }
                  >
                    Commentator
                  </span>
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                        "display": "block",
                        "fontSize": 14,
                        "muiPrepared": true,
                      }
                    }
                  />
                </div>
                <button
                  disabled={false}
                  href={undefined}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onMouseOut={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  onTouchTap={[Function]}
                  style={
                    Object {
                      "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                      "background": "none",
                      "border": 10,
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "cursor": "pointer",
                      "display": "inline-block",
                      "fontFamily": "Roboto, sans-serif",
                      "fontSize": 0,
                      "fontWeight": "inherit",
                      "height": 48,
                      "margin": "auto",
                      "muiPrepared": true,
                      "outline": "none",
                      "overflow": "visible",
                      "padding": 12,
                      "position": "absolute",
                      "right": 4,
                      "textDecoration": "none",
                      "top": 0,
                      "transform": "translate(0, 0)",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "verticalAlign": null,
                      "width": 48,
                    }
                  }
                  tabIndex={0}
                  type="button"
                >
                  <div
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchStart={[Function]}
                  >
                    <svg
                      onMouseEnter={[Function]}
                      onMouseLeave={[Function]}
                      style={
                        Object {
                          "color": "rgba(0, 0, 0, 0.87)",
                          "display": "inline-block",
                          "fill": "currentColor",
                          "height": 24,
                          "muiPrepared": true,
                          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                          "userSelect": "none",
                          "width": 24,
                        }
                      }
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M7.41 7.84L12 12.42l4.59-4.58L18 9.25l-6 6-6-6z"
                      />
                    </svg>
                  </div>
                </button>
              </div>
            </div>
          </div>
          <div
            className="search-tool-card"
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                "backgroundColor": "#ffffff",
                "borderRadius": "2px",
                "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
                       0 1px 4px rgba(0, 0, 0, 0.12)",
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "fontFamily": "Roboto, sans-serif",
                "muiPrepared": true,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "zIndex": 1,
              }
            }
          >
            <div
              style={
                Object {
                  "paddingBottom": 0,
                }
              }
            >
              <div
                className="card-header"
                onTouchTap={[Function]}
                style={
                  Object {
                    "boxSizing": "border-box",
                    "cursor": "pointer",
                    "fontFamily": "Proxima N W01 At Smbd",
                    "fontSize": "14px",
                    "fontWeight": "bold",
                    "muiPrepared": true,
                    "padding": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "whiteSpace": "nowrap",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "display": "inline-block",
                      "muiPrepared": true,
                      "paddingRight": "90px",
                      "verticalAlign": "top",
                      "whiteSpace": "normal",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.87)",
                        "display": "block",
                        "fontSize": 15,
                        "muiPrepared": true,
                      }
                    }
                  >
                    Reference
                  </span>
                  <span
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.54)",
                        "display": "block",
                        "fontSize": 14,
                        "muiPrepared": true,
                      }
                    }
                  />
                </div>
                <button
                  disabled={false}
                  href={undefined}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onMouseOut={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  onTouchTap={[Function]}
                  style={
                    Object {
                      "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                      "background": "none",
                      "border": 10,
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "cursor": "pointer",
                      "display": "inline-block",
                      "fontFamily": "Roboto, sans-serif",
                      "fontSize": 0,
                      "fontWeight": "inherit",
                      "height": 48,
                      "margin": "auto",
                      "muiPrepared": true,
                      "outline": "none",
                      "overflow": "visible",
                      "padding": 12,
                      "position": "absolute",
                      "right": 4,
                      "textDecoration": "none",
                      "top": 0,
                      "transform": "translate(0, 0)",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "verticalAlign": null,
                      "width": 48,
                    }
                  }
                  tabIndex={0}
                  type="button"
                >
                  <div
                    onMouseDown={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchStart={[Function]}
                  >
                    <svg
                      onMouseEnter={[Function]}
                      onMouseLeave={[Function]}
                      style={
                        Object {
                          "color": "rgba(0, 0, 0, 0.87)",
                          "display": "inline-block",
                          "fill": "currentColor",
                          "height": 24,
                          "muiPrepared": true,
                          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                          "userSelect": "none",
                          "width": 24,
                        }
                      }
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M7.41 7.84L12 12.42l4.59-4.58L18 9.25l-6 6-6-6z"
                      />
                    </svg>
                  </div>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
      <header>
        <div />
      </header>
    </div>
  </div>
  
</div>
`;
